phoenix-core/src/it/java/org/apache/phoenix/end2end/index/LocalIndexIT.java

```diff
@@ -727,7 +727,7 @@ public void testLocalIndexScanAfterRegionSplit() throws Exception {
             assertTrue(rs.next());
             
             HBaseAdmin admin = driver.getConnectionQueryServices(getUrl(), TestUtil.TEST_PROPERTIES).getAdmin();
-            for (int i = 1; i < 5; i++) {
+            for (int i = 1; i < 2; i++) {
```

Comment:
Any reason to go from 5 to 2 here? Maybe get rid of the loop then? Does it decrease the amount of testing being done?